bridgesample/src/main/java/com/livefront/bridgesample/scenario/fragment/LargeDataFragment.kt

```diff
@@ -40,15 +56,15 @@ class LargeDataFragment : BridgeBaseFragment() {
         fun getArguments(
             fragment: LargeDataFragment
         ): LargeDataArguments = fragment
-                .arguments!!
```

Comment:
Let's make sure we undo all this formatting.

Reply:
Done.